test/fail_compilation/mixin.d

```diff
@@ -0,0 +1,26 @@
+// REQUIRED_ARGS: -mixin=test.mixin
```

Comment:
You can use variables here too, e.g. 
https://github.com/dlang/dmd/blob/master/test/compilable/json.d

Reply:
The trouble is that the file name is also in the tested output. Can this be replaced aswell?